bin/docker-setup.sh

```diff
@@ -18,7 +18,11 @@ redirect_output() {
 # https://hub.docker.com/_/wordpress#running-as-an-arbitrary-user
 cli()
 {
-	redirect_output docker run -it --env-file default.env --rm --user xfs --volumes-from $WP_CONTAINER --network container:$WP_CONTAINER wordpress:cli "$@"
+	INTERACTIVE=''
```

Comment:
Maybe this could have been a separate PR 🤷  just to keep things single-scoped

Reply:
Sounds good! :+1: I'll create a different PR for this.